frontend/src/main/components/Commons/CommonsForm.js

```diff
@@ -119,6 +119,25 @@ function CommonsForm({ initialCommons, submitAction, buttonLabel = "Create" }) {
           {errors.startingDate?.message}
         </Form.Control.Feedback>
       </Form.Group>
+      <Form.Group className="mb-3">
+        <Form.Label htmlFor="degradationRate">Degradation Rate</Form.Label>
+        <Form.Control
+          data-testid={`${testid}-degradationRate`}
+          id="degradationRate"
+          type="number"
+          step="0.01"
+          isInvalid={!!errors.degradationRate}
+          {...register("degradationRate", {
+            valueAsNumber: true,
+            required: "Degradation Rate is required",
+            min: { value: 0.01, message: "Degration Rate must be positive number between 0 and 1" },
```

Comment:
We'd like this to be any positive number, not just between 0 and 1; it will be treated as a percentage.  (This is new information from Mattanjah Devries)

Can we change  the step to 0.001, the min to 0.001, and remove the max?

Reply:
Yes, I will work on it.